### DISCUSSION TITLE
Transaction amount on the chain is lower than the amount I sent through command line chia wallet send???

### DISCUSSION THREAD

**Discussion by Proulx-S** [Chia-Network/chia-blockchain#8345] [Support]
Hello,
I made the following transaction to myself, for 0.0001 XCH with 0 fee:
`$ chia wallet send -a 0.0001 -m 0 -t xxxxxxxxxxx -i 1`
 and it later confirmed just fine:
```
$ chia wallet get_transaction -f 361947034 -tx xxxxxxxxxxxx
Transaction xxxxxxxxxxx
Status: Confirmed<br>
Amount: 0.0001 xch
To address: xxxxxxxxxxxxx
Created at: 2021-09-05 16:18:34
```

But. Looking at the transaction in another way:
```
$ chia wallet get_transactions
...
Transaction xxxxxxxxxxxxxx
Status: Confirmed
Amount: 0.000097922202 xch
To address: xxxxxxxxxxxx
Created at: 2021-09-05 17:55:48
```
shows more decimal digits, and suggests the amount sent is actually 2% lower than the one I asked to be sent!

I also checked on chiaexplorer.com, but it rounds up to 0.0001.

What's up ?

The only reason I noticed this is because my third party wallet has a minimum deposit amount of 0.0001 XCH, and my 0.0001 XCH (or 0.000097922202 XCH) transaction is not showing up on the third party app.

Thanks for any help :-)
Sébastien

**Comment by BrandtH22** [comment]
@Proulx-S , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - without being able to review the onchain records myself (would need the coinIDs to do so) it appears that a fee might have been included in the transaction which would have accounted for the other 2%.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
